code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -1408,7 +1396,7 @@ func (sc *StorageSmartContract) canceledPassRates(alloc *StorageAllocation,
 			//if c.Responded || c.AllocationID != alloc.ID {
 			//	continue // already accepted, already rewarded/penalized
 			//}
-			var expire = oc.CreatedAt + toSeconds(ba.Terms.ChallengeCompletionTime)
+			var expire = oc.CreatedAt + toSeconds(alloc.MaxChallengeCompletionTime)
```

Comment:
we can get the MaxChallangeCompletionTime from gloabl config, so remove the alloc.MaxChallengeCompletionTime too

Reply:
fixed